Bottom: 214b93950c7f7ac964bd78189d089eb77fe3af70
Top:    c3b40b812391ba1713da5a80ce38ee88cc53c305
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 16:11:16 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3e11b01..c530d66 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -332,13 +332,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
